### PR TITLE
Dlib Force CMake flags

### PR DESCRIPTION
**Pull request by r4ghu**
This PR fixes the submodule update for the new `stream-labs/dlib` version. Our current facemask-plugin repo has a dlib submodule referenced to `https://github.com/stream-labs/dlib/tree/0b8dc886baf1d5786666ce64cd2de57327c96ca3` which leads to an empty github page. This resulted in errors while updating submodules. When I tried to delete the repo and clone it again the following error occured:
```
error: Server does not allow request for unadvertised object 0b8dc886baf1d5786666ce64cd2de57327c96ca3
Fetched in submodule path 'thirdparty/dlib', but it did not contain 0b8dc886baf1d5786666ce64cd2de57327c96ca3. Direct fetching of that commit failed.
```

So, I referenced it to the correct dlib handle and it has been observed that the new DLIB has some new flags to set. I forced them to `OFF` to ease the build of facemask-plugin.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
-[submodule "dlib"]
-	path = thirdparty/dlib
-	url = https://github.com/stream-labs/dlib.git
 [submodule "freetype2"]
 	path = thirdparty/freetype2
 	url = https://github.com/stream-labs/freetype2.git
+[submodule "thirdparty/dlib"]
+	path = thirdparty/dlib
+	url = https://github.com/stream-labs/dlib
```
Comment (brankalakic) [inline .gitmodules]: Thanks for drawing attention to broken submodule. It was not about the url, I made a mistake when updating it. I fixed it now. See if you can update your submodules and have it work now and let me know if there is still an issue

File: CMakeLists.txt [hunk 2]
```diff
@@ -9,6 +9,13 @@ set(DLIB_NO_GUI_SUPPORT OFF CACHE STRING "DLib GUI support" FORCE)
 set(DLIB_GIF_SUPPORT OFF CACHE STRING "DLib GIF support" FORCE)
 set(DLIB_JPEG_SUPPORT OFF CACHE STRING "DLib JPEG support"  FORCE)
 set(DLIB_PNG_SUPPORT OFF CACHE STRING "DLib PNG support"  FORCE)
+set(DLIB_USE_BLAS OFF CACHE STRING "DLib BLAS support"  FORCE)
+set(DLIB_USE_LAPACK OFF CACHE STRING "DLib LAPACK support"  FORCE)
+set(DLIB_USE_MKL_FFT OFF CACHE STRING "DLib MKL FFT support"  FORCE)
+set(DLIB_USE_MKL_SEQUENTIAL OFF CACHE STRING "DLib MKL Sequential support"  FORCE)
+set(DLIB_USE_MKL_WITH_TBB OFF CACHE STRING "DLib MKL with TBB support"  FORCE)
+
+
```
Comment (brankalakic) [inline CMakeLists.txt:18]: Is this for faster building of dlib?

Comment (r4ghu) [reply CMakeLists.txt:18]: Yup

Comment (r4ghu) [reply CMakeLists.txt:18]: Fixed the .gitmodules link. These CMake Flags have to be merged to master to default certain options in DLIB while building.

**Comment by r4ghu** [comment]
Updated the PR. The only changes are now CMake Flags for the current DLIB.

**Pull request merged**